client/state/index.js

```diff
@@ -136,6 +136,7 @@ export function createReduxStore( initialState = {} ) {
 		isBrowser && require( './happychat/middleware.js' ).default(),
 		isBrowser && require( './analytics/middleware.js' ).analyticsMiddleware,
 		isBrowser && require( './data-layer/wpcom-api-middleware.js' ).default,
+		isBrowser && require( './data-layer/extensions-middleware.js' ).default,
```

Comment:
do we need to browser-check extensions? could there be other contexts where we want them included?